src/permutedims/permutedims.jl

```diff
@@ -28,29 +23,55 @@ function fusiontensor_permutedims(
   return fusiontensor_permutedims(ft, biperm)
 end
 
-function fusiontensor_permutedims(ft, biperm::BlockedPermutation{2})
+function fusiontensor_permutedims!(
+  ftdst, ftsrc, new_codomain_indices::Tuple, new_domain_indices::Tuple
+)
+  biperm = permmortar((new_codomain_indices, new_domain_indices))
```

Comment:
```suggestion
  ftdst, ftsrc, new_codomain_dims::Tuple, new_domain_dims::Tuple
)
  biperm = permmortar((new_codomain_dims, new_domain_dims))
```
That's more consistent with Base Julia notation.